stake_version_intervals.go

```diff
@@ -100,6 +102,7 @@ func AllStakeVersionIntervals(ctx context.Context, dcrdClient *rpcclient.Client,
 		}
 	}
 
+	svis.MinVoteVersion = activeNetParams.GenesisBlock.Header.StakeVersion
```

Comment:
Actually, after a second look, this really isn't quite right.  It should be the mine between this and 4 so that it works on all networks.

Reply:
```suggestion
	svis.MinVoteVersion = maxUint32(activeNetParams.GenesisBlock.Header.StakeVersion, 4)
```

Reply:
Another approach would be to just find it in the deployments when the max is found right in the same loop.  e.g.

```Go
	// Get min and max vote versions.
	var min = math.MaxUint32
	var max uint32
	for version := range activeNetParams.Deployments {
		if version < min {
			min = version
		}
		if version > max {
			max = version
		}
	}
```